### DISCUSSION TITLE
Bazarr Menu not readable

### DISCUSSION THREAD

**Discussion by gregeeh** [themepark-dev/theme.park#552] [Q&A] [answered]
Hi all,

Using the Stylus Method on Brave for Bazarr and the left menu is unreadable until it is selected.  Same with all Bazarr Themes.  No issues with Radarr, Sonarr, Prowlarr, Deluge etc.

https://i.imgur.com/yoEMaZw.jpeg

Bazarr version is:  1.4.2

**Comment by gregeeh** [comment, marked as answer]
For anyone else with the same issue, you must set the UI Theme in Bazarr to Dark for TP to work correctly.

**Reply by GilbN** [reply]
I'll update the docs
